frontend/src/components/City.js

```diff
@@ -31,20 +25,18 @@ class City extends React.Component {
       left: cssLeft
     };
 
-    let owned = false;
     let selected = "";
-    if (this.state.clicked === true) {
+    if (this.props.selected === true) {
```

Comment:
nit: do you need the `===`? Isn't this the same as writing `if (this.props.selected === true)`